build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/views/types_module.rb

```diff
@@ -253,23 +262,34 @@ def initialize(options)
         # @return [String, nil]
         attr_accessor :documentation
 
+        # @return [Array<Symbol>]
+        attr_accessor :sensitive_params
+
         # @return [Boolean]
         def empty?
           @empty
         end
 
+        # @return [Boolean]
+        def sensitive_params?
+          @sensitive_params.any?
```

Comment:
I don't think this is a problem - but if sensitive_params isn't set at all on line 247, this will give an error.  To be safe I'd suggest having fetch have a default of []?